.release/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Should this file be checked in?

Reply:
Another good question. It contains a history of all the builds, but if we're going to build the AMIs on a box that doesn't have git commit access, then it doesn't make sense to check it in.

Reply:
Even if the build process has commit access, is there a benefit to having the full history of all the builds checked in?